guide/popular-topics/permissions-extended.md

```diff
@@ -21,49 +21,49 @@ Due to this system, you cannot deny base permissions. If you grant `SEND_MESSAGE
+`KickMembers`, `BanMembers`, `Administrator`, `ManageChannels`, `ManageGuild`, `ManageMessages`, `ManageRoles`, `ManageWebhooks`, `ManageThreads`, and `ManageEmojisAndStickers`.
 
 ## Implicit permissions
 
 Some Discord permissions apply implicitly based on logical use, which can cause unwanted behavior if you are not aware of this fact.
 
-The prime example for implicit permissions is `VIEW_CHANNEL`. If this flag is missing in the final permissions, you can't do anything on that channel. It makes sense, right? If you can't view the channel, you can't read or send messages in it, set the topic, or change its name.
+The prime example for implicit permissions is `ViewChannel`. If this flag is missing in the final permissions, you can't do anything on that channel. It makes sense, right? If you can't view the channel, you can't read or send messages in it, set the topic, or change its name.
 The library does not handle implicit permissions for you, so understanding how the system works is vital for you as a bot developer.
 
-Let's say you want to send a message to a channel. To prevent unnecessary API calls, you want to make sure your bot's permissions in this channel include `SEND_MESSAGES` (more on how to achieve this [here](/popular-topics/permissions.md#checking-for-permissions)). The check passes, but you still can't send the message and are greeted with `DiscordAPIError: Missing Access`.
+Let's say you want to send a message to a channel. To prevent unnecessary API calls, you want to make sure your bot's permissions in this channel include `SendMessages` (more on how to achieve this [here](/popular-topics/permissions.md#checking-for-permissions)). The check passes, but you still can't send the message and are greeted with `DiscordAPIError: Missing Access`.
 
-This error means your bot is missing `VIEW_CHANNEL`, and as such, can't send messages either.
+This error means your bot is missing `ViewChannel`, and as such, can't send messages either.
 
-One possible scenario causing this: the channel has permission overwrites for the default role `@everyone` to grant `SEND_MESSAGES` so everyone who can see the channel can also write in it, but at the same time has an overwrite to deny `VIEW_CHANNEL` to make it only accessible to a subset of members.
+One possible scenario causing this: the channel has permission overwrites for the default role `@everyone` to grant `SendMessages` so everyone who can see the channel can also write in it, but at the same time has an overwrite to deny `VIEW_CHANNEL` to make it only accessible to a subset of members.
 
-As you only check for `SEND_MESSAGES`, the bot will try to execute the send, but since `VIEW_CHANNEL` is missing, the API denies the request.
+As you only check for `SendMessages`, the bot will try to execute the send, but since `VIEW_CHANNEL` is missing, the API denies the request.
```

Comment:
```suggestion
As you only check for `SendMessages`, the bot will try to execute the send, but since `ViewChannel` is missing, the API denies the request.
```